qatrfm/cli.py

```diff
@@ -76,83 +108,61 @@ def find_tf_file(basedir):
 @click.option('--no-clean', 'no_clean', is_flag=True,
               help="Don't clean the environment when the tests finish. "
               "This is useful for debug and troubleshooting.")
-def cli(test, path, tfvar, snapshots, no_clean):
+def cli(test, tfvar, snapshots, no_clean):
     """ Create a terraform environment and run the test(s)"""
 
     logger = QaTrfmLogger.getQatrfmLogger(__name__)
```

Comment:
shall we move this out of this method, so that we can also use it above in find_corresponding_tf_file?